sos/__init__.py

```diff
@@ -20,6 +20,7 @@
 import sys
 
 from argparse import ArgumentParser
+from pathlib import Path
```

Comment:
I think this is not further required import (while/though I am curious why Travis didnt detect it as superfluous).